.travis.yml

```diff
@@ -1,27 +1,32 @@
 language: c
-dist: trusty
-sudo: required
-before_install:
-    - sudo add-apt-repository -y "deb http://archive.ubuntu.com/ubuntu/ trusty main universe"
-    - sudo apt-get update -qq
-    - sudo apt-get install libgcrypt20 liblzma5 libselinux1
-    - wget "http://archive.ubuntu.com/ubuntu/ubuntu/pool/main/s/systemd/libsystemd0_219-7ubuntu3_amd64.deb"
-    - wget "http://archive.ubuntu.com/ubuntu/ubuntu/pool/main/s/systemd/libsystemd-dev_219-7ubuntu3_amd64.deb"
-    - sudo dpkg -i libsystemd0_219-7ubuntu3_amd64.deb libsystemd-dev_219-7ubuntu3_amd64.deb
-script: bash -ex travis.sh
+sudo: false
+services:
+  - docker
+script:
```

Comment:
I think the echo commands should be quoted to give more assurance that we understand what goes into the final file.